index.html

```diff
@@ -18,22 +18,125 @@
     </header>
     <main class="main-section">
       <section class="content">
-          <h1>I’m Muhammed Glad to see you!</h1>
-          <p id="primary-text">
-            I’m a software developer! I can help you build a product , feature
-            or website Look through some of my work and experience! If you like
-            what you see and have a project you need coded, don’t hestiate to
-            contactl me.
-          </p>
-          <p class="social-media">LET’S CONNECT</p>
-          <ul class="icons">
-            <li><img src="./assets/Vector.svg" alt="twitter" /></li>
-            <li><img src="./assets/Linkedin icon.svg" alt="linkedin" /></li>
-            <li><img src="./assets/Vector (2).svg" alt="m" /></li>
-            <li><img src="./assets/Vector (1).svg" alt="github" /></li>
-            <li> <img src="./assets/Vector (3).svg" alt="s" /></li>
-          </ul>
-        </div>
+        <h1>I’m Muhammed Glad to see you!</h1>
+        <p id="primary-text">
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contactl
+          me.
+        </p>
+        <p class="social-media">LET’S CONNECT</p>
+        <ul class="icons">
+          <li><img src="./assets/Vector.svg" alt="twitter" /></li>
+          <li><img src="./assets/Linkedin icon.svg" alt="linkedin" /></li>
+          <li><img src="./assets/Vector (2).svg" alt="m" /></li>
+          <li><img src="./assets/Vector (1).svg" alt="github" /></li>
+          <li><img src="./assets/Vector (3).svg" alt="s" /></li>
+        </ul>
+      </section>
+      <section class="portfolio-list-section">
+        <img src="./assets/Snapshoot Portfolio.png" alt="Snapshoot-1" />
+        <p class="project-title">Tonic</p>
```

Comment:
- [ ] I would like to recommend you use `<hx>` elements instead `<p>` tags for titles and subtitles, because it is a good practice and semantically correct :+1: